server/app/routes/v1/services/service_container_logs.rb

```diff
@@ -5,7 +5,7 @@
     r.is do
       scope = @grid_service.container_logs
 
-      scope = scope.where(name: r['container']) unless r['container'].nil?
```

Comment:
Our 1.0 API probably means that we must keep the existing `?container=` param. Add the `?instance=` as a new parameter?